src/account/default.ts

```diff
@@ -124,6 +158,49 @@ export class Account extends Provider implements AccountInterface {
     );
   }
 
+  public async declare(
+    { classHash, contract }: DeclareContractPayload,
+    transactionsDetail: InvocationsDetails = {}
+  ): Promise<DeclareContractResponse> {
+    const nonce = toBN(transactionsDetail.nonce ?? (await this.getNonce()));
+    let maxFee: BigNumberish = '0';
```

Comment:
can this be like:
```suggestion
    let maxFee: ZERO;
```
?

Reply:
```suggestion
    let maxFee = ZERO;
```
Do you mean this?

Reply:
Yeah, sure, if that is ok? Saw being ZERO on another place